### PR TITLE
OEUI-86: Implement updates to POST discontinuing an order

### PR DESCRIPTION
**Pull request by zeze111**
[OEUI-86: Implement updates to POST discontinuing an order](https://issues.openmrs.org/browse/OEUI-86)

### SUMMARY:
This PR includes additional updates to current implementation of posting a drug order to allow an order to be discontinued

### CODE REVIEW

File: app/js/components/orderEntry/ActiveOrders.jsx [hunk 1]
```diff
@@ -37,33 +37,42 @@ export class ActiveOrders extends React.Component {
...
         uuid,
+        careSetting: careSetting.uuid,
         drugName: drug.display,
         action: 'DISCONTINUE',
         dose,
-        dosingUnit,
-        frequency,
-        route,
+        drug: drug.uuid,
+        dosingUnit: (doseUnits && doseUnits.display) || doseUnits,
```
Comment (humanityjs) [inline app/js/components/orderEntry/ActiveOrders.jsx]: Hey. What is this line suppose to do? Just curious.

File: app/js/components/orderEntry/ActiveOrders.jsx [hunk 2]
```diff
@@ -37,33 +37,42 @@ export class ActiveOrders extends React.Component {
...
+        frequency: (frequency && frequency.display) || frequency,
```
Comment (humanityjs) [inline app/js/components/orderEntry/ActiveOrders.jsx]: Same here

File: app/js/components/orderEntry/ActiveOrders.jsx (hunk 1 above)
Comment (zeze111) [reply app/js/components/orderEntry/ActiveOrders.jsx]: it checks that there is a valid "value" in "doseUnits", if so then return "doseUnits.display" or else return whatever value is in "doseUnits", which can either be "null" or empty string " ".

File: app/js/components/orderEntry/ActiveOrders.jsx (hunk 2 above)
Comment (zeze111) [reply app/js/components/orderEntry/ActiveOrders.jsx]: Same as above

**Pull request merged**